pkgs/development/compilers/gambit/build.nix

```diff
@@ -83,15 +84,15 @@ gccStdenv.mkDerivation rec {
 
```

Comment:
/usr/local/opt/openssl@1.1/lib 3 lines above here does not match anything and should be removed or fixed.

Reply:
I think they match https://github.com/gambit/gambit/blob/6acd87dfa95bfca33082a431e72f023345dc07ee/configure.ac#L2041?

Reply:
Ok the second substitution doesn't match anything but the first one does.

Reply:
On inspecting the build logs, it was using openssl in `/usr/local/lib`. I have fixed the substitutions to make sure this doesn't happen.